src/log_outgoing_requests/admin.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib import admin
+
+from .models import OutgoingRequestsLog
+
+
+@admin.register(OutgoingRequestsLog)
+class OutgoingRequestsLogAdmin(admin.ModelAdmin):
+    fields = (
+        "hostname",
+        "path",
+        "params",
+        "query_params",
+        "status_code",
+        "method",
+        "response_ms",
+        "timestamp",
+        "trace",
+    )
+    readonly_fields = fields
+    list_display = fields
+    list_filter = ("method", "status_code", "hostname")
+    search_fields = ("path",)
+    date_hierarchy = "timestamp"
```

Comment:
Couple of cosmetics/UIX points:

- Add full URL readonly field to admin detail (for copy-paste convenience when debugging issues)
- Remove `trace` column form admin list (too large)
- Override admin `has_change_permission`/`has_add_permission` (not relevant here, but delete could be in case of sensitive data issue)
- Add more fields to search_fields (`params, query_params, hostname`)